### PR TITLE
Update Singularity-CRI user docs

### LINKED ISSUE #11: Update userdocs after beta2 release

### PR DESCRIPTION
**Pull request by sashayakovtseva**
Closes #11

### CODE REVIEW

File: docs/examples.rst [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+	kubectl apply -f examples/k8s/image-service.yaml
+
+To verify objects are indeed created you can do:
+
+.. code-block:: bash
+
+	kubectl get deploy && \
+	kubectl get svc
+
+If everything is fine you should be able to access the cats server through just created Kubernetes service.
```
Comment (jchavezlavalle) [inline docs/examples.rst]: change this line to " If everything is fine you should be able to access the cats server through the **Kubernetes service that has just been created** "

File: docs/examples.rst [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+There are couple URLs that where created for testing purposes, ``/cats/good`` is the one with cats.
+Don't question namings :)
```
Comment (jchavezlavalle) [inline docs/examples.rst]: is it okay to add emojis? :smile:

File: docs/examples.rst [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+.. code-block:: text
+
+	GET /books/<id>
+
+- Delete existing book
+
+.. code-block:: text
+
+	POST /books/<id>:delete
```
Comment (jchavezlavalle) [inline docs/examples.rst:125]: why isn't here a `DELETE` request instead of a `POST`? just curious question

File: docs/examples.rst [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+.. image:: img/darkflow.png
+
+To create a deployment and a service run the following:
+
+.. code-block:: bash
+
+	kubectl apply -f darkflow.yaml
+
+To verify objects are indeed created you can do:
```
Comment (jchavezlavalle) [inline docs/examples.rst]: Change this line to: "To verify **that** objects are ...."

File: docs/examples.rst [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+
+.. code-block:: bash
+
+	kubectl get deploy && \
+	kubectl get svc
+
+If everything is fine you should be able to access Darkflow UI that is exposed with darkflow-web
```
Comment (jchavezlavalle) [inline docs/examples.rst]: maybe add on `` notation words like Darkflow UI or darkflow-web

File: docs/installation.rst [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+--------------------
+Install Dependencies
+--------------------
+
+1) Install `git <https://git-scm.com/downloads>`_
+2) Install `Singularity 3.1+ <https://www.sylabs.io/guides/3.0/user-guide/installation.html>`_ with OCI support
+3) Install `Go 1.11+ <https://golang.org/doc/install>`_
+4) Install `inotify <http://man7.org/linux/man-pages/man7/inotify.7.html>`_ to enable GPU device plugin
+5) Install socat if you want to enable fort-forwarding, e.g
```
Comment (jchavezlavalle) [inline docs/installation.rst]: add socat on notation ``

File: docs/installation.rst [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+
+.. _remove-an-old-version:
+
+---------------------
+Remove an old version
+---------------------
+
+When you run ``sudo make install``, the command lists files as they are
+installed. They must be removed in order to completely remove Singularity-CRI from your host.
+For convenience we created uninstall command, so you can run the following to cleanup installation:
```
Comment (jchavezlavalle) [inline docs/installation.rst]: add uninstall under ```notation

File: docs/k8s.rst [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+
+If you are looking for trying Singularity-CRI locally, follow :ref:`local testing guide with Sykube <sykube>`.
+
+To fully enable Singularity support in Kubernetes cluster, Singularity-CRI should be installed
+on each Kubernetes node. However, one may choose to have a heterogeneous cluster with multiple container runtimes.
+In that case only dedicated Kubernetes nodes should be integrated, and no changes should be done to the rest.
+
+To make Kubernetes work with Singularity-CRI a couple of steps are needed:
+
+1) create Singularity-CRI service
```
Comment (jchavezlavalle) [inline docs/k8s.rst]: Add the numbered list notation on rst : 

#. create Singularity-CRI service
#. modify kubelet config
#. restart kubelet with new config

File: docs/k8s.rst [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+
+.. code-block:: bash
+
+	sudo systemctl enable sycri && \
+	sudo systemctl start sycri
+
+
+.. note::
+	Singularity-CRI service above uses default configuration and log level. You can modify both
+	of that if you wish, refer to :ref:`configuration section <configuration>`.
```
Comment (jchavezlavalle) [inline docs/k8s.rst]: change line: You can modify both of **them** if you wish...

File: docs/k8s.rst [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+
+.. note::
+	Latest Singularity plugin system is not stable and leads to panic when no HOME and GOPATH
```
Comment (jchavezlavalle) [inline docs/k8s.rst]: add env vars like `HOME` and `GOPATH` under the ```notation on rst

File: docs/k8s.rst [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+			   └─31927 /usr/local/bin/sycri
+
+	Jun 20 16:01:38 ubuntu-bionic systemd[1]: Started Singularity-CRI.
+	Jun 20 16:01:38 ubuntu-bionic sycri[1989]: I0620 16:01:38.980735    1989 network.go:112] Network configuration found: bridge
+	Jun 20 16:01:38 ubuntu-bionic sycri[1989]: I0620 16:01:38.993158    1989 main.go:193] Singularity-CRI server started on /var/run/singularity.sock
+	Jun 20 16:01:39 ubuntu-bionic sycri[1989]: I0620 16:01:39.095791    1989 device.go:97] Loading NVML
+	Jun 20 16:01:39 ubuntu-bionic sycri[1989]: E0620 16:01:39.096779    1989 device.go:99] Could not initialize NVML library: could not load NVML library
+	Jun 20 16:01:39 ubuntu-bionic sycri[1989]: W0620 16:01:39.097603    1989 main.go:209] GPU support is not enabled: unable to load: check libnvidia-ml.so.1 library and graphic drivers
+
+Optionally you may want to disable other runtime services, e.g. docker daemon.
```
Comment (jchavezlavalle) [inline docs/k8s.rst]: Add this on a note

File: docs/k8s.rst [hunk 12]
```diff
@@ -0,0 +1,137 @@
...
+
+Modify kubelet config
+---------------------
+
+Kubelet needs to be reconfigured so that it connects to Singularity-CRI.
+If you haven't change default config, the following will be enough:
```
Comment (jchavezlavalle) [inline docs/k8s.rst]: If you haven't **changed** ...

File: docs/local.rst [hunk 13]
```diff
@@ -0,0 +1,171 @@
...
+2. Configure it work with Singularity-CRI. Create `/etc/crictl.yaml` config file and add the following:
+
+.. code-block:: text
+
+	runtime-endpoint: unix:///var/run/singularity.sock
+	image-endpoint: unix:///var/run/singularity.sock
+	timeout: 10
+	debug: false
+
+For details on all options available see `crictl install page
```
Comment (jchavezlavalle) [inline docs/local.rst]: For details on all the available options see..

File: docs/local.rst [hunk 14]
```diff
@@ -0,0 +1,171 @@
...
+	make && \
+	sudo make install && \
+	sudo sycri
+
+
+-----------------------------
+Interact with Singularity-CRI
+-----------------------------
+
+We will walk through a basic examples of interaction with a running Singularity-CRI.
```
Comment (jchavezlavalle) [inline docs/local.rst]: change this line to : We will walk through basic examples of interaction with Singularity-CRI running.

File: docs/local.rst [hunk 15]
```diff
@@ -0,0 +1,171 @@
...
+Further assumed you are located in Singularity-CRI repository root.
```
Comment (jchavezlavalle) [inline docs/local.rst:49]: maybe change this line to: We will assume that you are located in ....

File: docs/local.rst [hunk 16]
```diff
@@ -0,0 +1,171 @@
...
+
+Running Nginx
+-------------
+
+1. Run a pod which exposes port 80:
```
Comment (jchavezlavalle) [inline docs/local.rst:54]: maybe add that this port is 80 from the default port opening from Nginx, for secure connections is port 443

File: docs/local.rst [hunk 17]
```diff
@@ -0,0 +1,171 @@
...
+
+.. code-block:: bash
+
+	sudo crictl runp examples/net-pod.json
```
Comment (jchavezlavalle) [inline docs/local.rst]: Add the `$` to all commands on code blocks

File: docs/local.rst [hunk 18]
```diff
@@ -0,0 +1,171 @@
...
+
+Unlike pods, containers should be explicitly started after creation:
+
+.. code-block:: bash
+
+	sudo crictl start <CONTAINER_ID>
+
+
+Verify nginx container is running by opening `localhost:80 <http://localhost:80>`_ in any browser.
+You should see nginx welcome page.
```
Comment (jchavezlavalle) [inline docs/local.rst]: You should see **the** Nginx welcome page.

File: docs/local.rst [hunk 19]
```diff
@@ -0,0 +1,171 @@
...
+	uid=0 gid=0 euid=0 egid=0
+	pid=30 ppid=0
+	envs=[LD_LIBRARY_PATH=/.singularity.d/libs SHLVL=1 MY_ANOTHER_VAR=is-awesome PS1=Singularity>  TERM=xterm PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin PWD=/ MY_CUSTOM_VAR=singularity-cri]
+	...
+
+
+Cleanup examples
+----------------
+
+The quickest way to cleanup is simply remove containing pods:
```
Comment (jchavezlavalle) [inline docs/local.rst]: The easiest/quickest way to do a cleanup is simply **by removing** containing pods:

**Review by jchavezlavalle — CHANGES_REQUESTED** [review]

File: docs/examples.rst (hunk 2 above)
Comment (sashayakovtseva) [reply docs/examples.rst]: Why not ? 😂

File: docs/examples.rst (hunk 3 above)
Comment (sashayakovtseva) [reply docs/examples.rst:125]: Honestly, I copied this API to make smth really quick when having first CRI demo. So I didn't change anything.

File: docs/local.rst (hunk 17 above)
Comment (sashayakovtseva) [reply docs/local.rst]: I intentionally removed them for easier copy-paste. Shouldn't have I?

File: docs/local.rst (hunk 16 above)
Comment (sashayakovtseva) [reply docs/local.rst:54]: I think for local examples there is not need in this details, port 80 will be used to smoke test CRI

**Pull request merged**